### ISSUE TITLE
Unable to create postgresflex_database specifiying an owner having hyphens in username

### ISSUE THREAD

**Issue by lordon** [stackitcloud/terraform-provider-stackit#487]
Terraform Version 1.9.3
Stackit Provider Version 0.26.1

When using terraform in order to create PostgreSQL Flex databases, it will end up in an inconsistent result when the specified owner username contains hyphens.

Working example from documentation:

```
resource "stackit_postgresflex_database" "example" {
  project_id  = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
  instance_id = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
  name        = "mydb"
  owner       = "myusername"
}
```

Non-working example:

```
resource "stackit_postgresflex_database" "example" {
  project_id  = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
  instance_id = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
  name        = "mydb"
  owner       = "my-username"
}
```

will produce the following error:

> Error: Provider produced inconsistent result after apply
>
> When applying changes to stackit_postgresflex_database.example, provider "provider[\"registry.terraform.io/stackitcloud/stackit\"]" produced an unexpected new value: .owner: was
> cty.StringVal("my-username"), but now cty.StringVal("\"my-username\"").
>
> This is a bug in the provider, which should be reported in the provider's own issue tracker.

**Comment by joaopalet** [comment]
Hi @lordon,

Thanks for opening the issue. I will be having a look and will provide a fix soon.

**Comment by GokceGK** [comment]
Hi @lordon, 

The fix was released in [v0.26.2](https://github.com/stackitcloud/terraform-provider-stackit/releases/tag/v0.26.2)
